libraries/AP_Networking/AP_Networking.cpp

```diff
@@ -214,10 +214,12 @@ void AP_Networking::announce_address_changes()
         return;
     }
 
+#if AP_HAVE_GCS_SEND_TEXT
     char ipstr[16];
```

Comment:
I assume you're doing this new AP_HAVE_GCS_SEND_TEXT wrap because it says the variable is unused. Seems excessive, maybe just change the line to: `char ipstr[16]; (void)ipstr; // cast void in case GCS_SEND_TEXT is empty like in the bootlader`

Reply:
i think the AP_HAVE_GCS_SEND_TEXT is neater, and allows you to avoid more code in a clearer fashion